src/client/index.js

```diff
@@ -0,0 +1,158 @@
+/* eslint-disable no-console */
+/* eslint-disable no-undef */
+const ws = new WebSocket('ws://localhost:8080');
```

Comment:
Ensure that the WebSocket server is running at 'ws://localhost:8080'. Otherwise, the connection will fail.